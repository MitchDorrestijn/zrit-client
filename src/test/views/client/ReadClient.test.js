import React from 'react';
import {shallow} from 'enzyme';
import toJSON from 'enzyme-to-json';
import ReadChauffeur from '../../../main/views/client/ReadClient';

const wrapper = shallow(<ReadChauffeur />);

test('<ReadClient /> should match snapshot', () => {
  expect(toJSON(wrapper)).toMatchSnapshot();
});

test('<ReadClient /> should render one <TextContent> component', () => {
  expect(wrapper.find('TextContent').length).toBe(1);
});

test('<ReadClient /> should render one <ClientTable> component', () => {
  expect(wrapper.find('ClientTable').length).toBe(1);
});

test('<TextContent /> in <ReadClient /> should have a title of Cliënten beheren', () => {
  const childWrapper = wrapper.find('TextContent');
  expect(childWrapper.prop('title')).toBe("Cliënten beheren");
});
