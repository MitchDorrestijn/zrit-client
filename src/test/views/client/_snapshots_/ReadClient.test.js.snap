// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ReadClient /> should match snapshot 1`] = `
<div>
  <TextContent
    title="Cliënten beheren"
  >
    Hieronder kunt u cliënten beheren. Druk op het rondje vooraan de tabel om een record te selecteren, vervolgens kunt u het record bewerken door op de knop 'bewerken' te drukken. Om een nieuw item toe te toevoegen drukt u op de knop 'Toevoegen'.
  </TextContent>
  <ClientTable />
</div>
`;
